packages/maskbook/src/plugins/ITO/types.ts

```diff
@@ -19,14 +19,13 @@ export interface JSON_PayloadInMask {
     chain_id: ChainId
     start_time: number
     end_time: number
+    unlock_time: number
+    qualification_address: string
     creation_time: number
     token: EtherTokenDetailed | ERC20TokenDetailed
     exchange_amounts: string[]
     exchange_tokens: (EtherTokenDetailed | ERC20TokenDetailed)[]
-    // @deprecated
```

Comment:
Please keep this. Learn more: https://github.com/DimensionDev/Maskbook/pull/2777#discussion_r608476872